src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -617,14 +613,24 @@ def forward(
         input_ids=None,
         past_key_values=None,
         attention_mask=None,
-        position_ids=None,
         head_mask=None,
         inputs_embeds=None,
         use_cache=None,
         output_attentions=None,
         output_hidden_states=None,
         return_dict=None,
+        **deprecated_arguments
     ) -> Union[Tuple[torch.Tensor], BaseModelOutputWithPastAndCrossAttentions]:
+        if deprecated_arguments.pop("position_ids", False) is not False:
+            # `position_ids` could have been `torch.Tensor` or `None` so defaulting pop to `False` allows to detect if users were passing explicitly `None`
```

Comment:
That's probably a bit overkill 😅

Reply:
Well I just wanted to make sure to send the message to everyone :D otherwise users passing `position_ids=None` are never going to see anything and their code will crash in v5.0.0